.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -20,23 +20,44 @@ body:
     attributes:
       value: |
         ## Required information
+
   - type: textarea
     id: description
     attributes:
       label: Description
-      description: Please provide a clear, concise and descriptive explanation of what the bug is. Include screenshots or a video if needed. Tell us what were you expecting to happen instead of what is happening now.
+      description: | 
+        Please provide a clear, concise and descriptive explanation of what the bug is. 
+        Make sure you tell us:
+        - [ ] What is the **actual** behavior
+        - [ ] What is the **expected** behavior
```

Comment:
Have you checked how does it look like in the final rendered issue template? (You can check it on some private repository).